DavidGonzalezZumaya/Collections/src/JavaMap/JavaMap.java

```diff
@@ -0,0 +1,36 @@
+package JavaMap;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class JavaMap {
+
+    public static void main(String []argh)
+    {
+        Scanner in = new Scanner(System.in);
+        Map<String, Integer> phoneDirectory = new HashMap<>();
```

Comment:
It is recommended that the name of the collections and maps start with singular name and them the name of the interface, example: numberList